Tests/Fixtures/FixtureFactory.php

```diff
@@ -56,6 +56,7 @@ final class FixtureFactory
     public const FIXTURE_SCOPE_SECOND = 'rock';
 
     public const FIXTURE_USER = 'user';
+    public const FIXTURE_PASSWORD = 'password';
```

Comment:
@HypeMC I don't see this being used anywhere?

Reply:
@X-Coder264 It was add with https://github.com/trikoder/oauth2-bundle/pull/18/files#diff-a87d1acb0e93f0f5a6dd7abd960527e3 , the tests pass without it, but I figured having it doesn't hurt.